sdk/packages/selector-polkadot/src/adapter.ts

```diff
@@ -370,6 +371,15 @@ export class NightlyConnectAdapter implements Injected {
 
       this._chosenMobileWalletName = walletName
 
+      if(walletName === 'subwallet-js' && window){
```

Comment:
This line should be `typeof window !== 'undefined' && walletName === 'subwallet-js'`